src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -48,6 +48,23 @@ internal void PopulateAutoCompleteCandidates()
             });
         }
 
+        internal void SearchAutoCompleteCandidates(string input)
+        {
+            if (PortViewModel == null) return;
+
+            var searchElements = GetMatchingNodes();
+            FilteredResults = searchElements.Select(e =>
+            {
+                if (e.Name.Contains(input))
```

Comment:
it also seems not to use the regular search algorithm.... that might be a good thing.

Reply:
I am not clear on this. The regular search algorithm that we use for library search of in-canvas search?